bin/build-github-pages-redirects.py

```diff
@@ -0,0 +1,47 @@
+"""
+Builds the Github Pages for this repo, which now only redirects to
+drafts.csswg.org
+"""
+
+import glob
+import re
+import os
+import os.path
+
+paths_to_build = set(["/", "/css-snapshot/"])
+
+for file in glob.glob("**/*.bs"):
+    if file.endswith("/Overview.bs"):
+        folder = "/" + file.removesuffix("Overview.bs")
+        paths_to_build.add(folder)
+
+        # If the folder ends with a version number, remove it to build
+        # the unversioned spec as well.
+        # We make sure to exclude css-20XX snapshots.
```

Comment:
Oh, I just noticed that https://drafts.csswg.org/css/ does serve the 2020 snapshot (not the current one). Should https://w3c.github.io/csswg-drafts/css redirect to drafts.csswg.org as well, then?